docs/howto_sysconf_watch.md

```diff
@@ -138,4 +142,30 @@ The below (non-proportional!) graph shows when certificates are rotated.
   certificate will be rotated for a new one.
 
 If the rotated certificate is a CA, all of the certificates it signed get rotated
-as well.
\ No newline at end of file
+as well.
+
+## Firewall Changes
+
+Reload the firewall rules with the following command to trigger the MicroShift service restart.
+
+```bash
+sudo firewall-cmd --reload
+```
+
+Firewall reload action flushes the existing iptable configurations which results in failed network traffic. <br>
+Run the `journalctl` command to verify that the service was restarted. The logs should contain restart and startup messages.
+
+```
+Dec 11 08:23:30 localhost.localdomain microshift[168331]: sysconfwatch-controller I1211 08:23:30.897394  168331 iptables.go:590] iptables canary mangle/MICROSHIFT-SYSCONF-CANARY deleted
+Dec 11 08:23:30 localhost.localdomain microshift[168331]: sysconfwatch-controller I1211 08:23:30.897919  168331 iptables.go:590] iptables canary mangle/MICROSHIFT-SYSCONF-CANARY deleted
+Dec 11 08:23:30 localhost.localdomain microshift[168331]: sysconfwatch-controller I1211 08:23:30.914732  168331 iptables.go:616] Reloading after iptables flush
+Dec 11 08:23:30 localhost.localdomain microshift[168331]: sysconfwatch-controller W1211 08:23:30.914772  168331 sysconfwatch_linux.go:112] iptables flush is detected, restarting MicroShift
+Dec 11 08:23:30 localhost.localdomain systemd[1]: microshift.service: Succeeded.
+Dec 11 08:23:30 localhost.localdomain systemd[1]: microshift.service: Consumed 28.612s CPU time
+Dec 11 08:23:31 localhost.localdomain systemd[1]: microshift.service: Service RestartSec=100ms expired, scheduling restart.
+Dec 11 08:23:31 localhost.localdomain systemd[1]: microshift.service: Scheduled restart job, restart counter is at 1.
+Dec 11 08:23:31 localhost.localdomain systemd[1]: Stopped MicroShift.
+Dec 11 08:23:31 localhost.localdomain systemd[1]: microshift.service: Consumed 28.612s CPU time
+Dec 11 08:23:31 localhost.localdomain systemd[1]: Starting MicroShift...
+Dec 11 08:23:42 localhost.localdomain microshift[171333]: ??? I1211 08:23:42.114965  171333 run.go:118] Starting MicroShift
+```
```

Comment:
Can we present the output of `sudo journalctl -xu microshift | egrep 'Starting MicroShift|restarting MicroShift'` command to be consistent with other restart cases?

Reply:
sure, done.